cspatients/models.py

```diff
@@ -16,6 +16,20 @@ def __str__(self):
 
 
 class PatientEntry(models.Model):
+    ELECTIVE = 1
+    COLD = 2
+    HOT_YELLOW = 3
+    HOT_ORANGE = 4
+    IMMEDIATE = 5
+
+    URGENCY_CHOICES = (
+        (ELECTIVE, "Elective"),
+        (COLD, "Cold"),
+        (HOT_YELLOW, "Hot"),
+        (HOT_ORANGE, "Hot"),
```

Comment:
Should these have the same key?

Reply:
Yeah hot yellow and hot orange. hehe
![hot_hot](https://user-images.githubusercontent.com/21120463/58707737-086a9500-83b6-11e9-810f-914433c2a7ce.png)